packages/modules/delete-itwin/src/components/DeleteITwin.scss

```
@@ -18,7 +18,7 @@
   }
 }
 
-.iac-delete-itwin-overlay-container {
+.iac-delete-iTwin-overlay-container {
```

dont camel case css class names
```suggestion
.iac-delete-itwin-overlay-container {
```

---

Done